sdk/python/feast/infra/registry/base_registry.py

```diff
@@ -99,6 +99,28 @@ def get_entity(self, name: str, project: str, allow_cache: bool = False) -> Enti
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def tag_entity(
```

Comment:
Just a thought, I see the default implementation is duplicated in all registry implementations. Can we mark it is non abstract method and override wherever is necessary. I don't know if we are breaking any code guidelines from feast with this type of implementation.

Reply:
I might be wrong, but I fear there are different hierarchies in the actual implementations preventing to have a common definition. @tokoko ?

Reply:
I think it could work regardless, but it's a moot point, we agreed to close the PR, see #4459.